wsmaster/che-core-api-workspace/src/main/resources/schema/1.0.0/devfile.json

```diff
@@ -35,6 +35,7 @@
     "apiVersion": {
       "type": "string",
       "title": "Devfile API Version",
+      "pattern": "1\\.0\\.0",
```

Comment:
imo `const` or `enum` should be more clear for reader here, but up2u

Reply:
better. fixed